api/user/author/model.py

```diff
@@ -17,6 +18,14 @@ class Author(UserMixin, db.Model):
     password: str = db.Column("password", db.String(64), nullable=False)
     github: str = db.Column("github", db.Text, nullable=True)
     profile_image: str = db.Column("profile_image", db.Text, default="")
+    follows = db.relationship(
+        "Author",
+        secondary=follows_table,
+        primaryjoin=id == follows_table.c.followed_id,
+        secondaryjoin=id == follows_table.c.follower_id,
+        lazy="dynamic",
+    )  # only load followers when requested
+    non_local_follows = db.relationship("NonLocalFollower", lazy="dynamic")
```

Comment:
I will take your word this just works ;)
Assuming the lazy=dynamic is what is lazy loading things here?

Reply:
I read into it. lazy=dynamic means it will only run the query to fetch the followers when we ask for it. I looked at the queries in debugger and it seems to be doing what it promised. lazy=join would load things by default. not ideal loading data we don't need.